ext/reflection/php_reflection.c

```diff
@@ -3856,14 +3847,14 @@ ZEND_METHOD(ReflectionClass, getStaticPropertyValue)
 	zend_string *name;
 	zval *prop, *def_value = NULL;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "S|z", &name, &def_value) == FAILURE) {
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "S|z!", &name, &def_value) == FAILURE) {
```

Comment:
Hm, this doesn't look right to me. Doesn't this mean that `getStaticPropertyValue("foo", null)` will throw now, if the property doesn't exist?

Reply:
Oh, yes, you're right. I'll get rid of this change.